examples/p5Scores.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+	<title>SHAPES AND SCORES</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+
+	<script type="text/javascript" src="./scripts/p5.js"></script>
+	<script type="text/javascript" src="http://cdn.tonejs.org/r6/p5.Tone.js"></script>
+	<script type="text/javascript" src="./scripts/jquery.min.js"></script>
+	<script type="text/javascript" src="./scripts/Interface.js"></script>
+
+	<link rel="stylesheet" type="text/css" href="./style/examples.css">
+
+	<script type="text/javascript">
+		// jshint ignore: start
+	</script>
+
+</head>
+<body>	
+	<div id="Content" class="FullScreen">
+		<div id="Title">Scoring Events</div>
+		<div id="Explanation">
+			Scores let you to schedule a note on a particular "channel". Then use <code>Tone.Note.route</code>
```

Comment:
Actually this kind of score stuff is deprecated since r6. (also MidiToScore has moved to another repo entirely). I'm using Tone.Part instead. 

This example might not be less relevant now since there's already another example in the r7 which explains the MIDI pipeline called [jump.html](https://github.com/Tonejs/Tone.js/blob/dev/examples/jump.html)